.github/workflows/neofs.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish to NeoFS
+on:
+  push:
+    branches: [ master ]
+
+env:
```

Comment:
Remove the variable definitions from here - they will be set by @roman-khimov in the repository settings.